sidebars.yaml

```diff
@@ -31,11 +31,14 @@
                 - page: docs/use-cases/tokenization/nftoken-marketplace.md
                 - page: docs/use-cases/tokenization/authorized-minter.md
                 - page: docs/use-cases/tokenization/digital-artist.md
+            - page: docs/use-cases/tokenizations.page.tsx
+              label: Tokenizations
```

Comment:
Isn't this page is meant to replace the `use-cases/tokenization/index.md` landing page (line 24)?